### PR TITLE
Cheetah Initial 9.3 release

### PR DESCRIPTION
**Pull request by osm1019**

### CODE REVIEW

**Comment by osm1019** [comment]
initial request for cheetah

File: changelog_cheetah.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Highlights & Device Specific Changes:
+Build type: weekly
+Device: Google Pixel 7 Pro (cheetah
```
Comment (gwolf2u) [inline changelog_cheetah.txt:8]: fix closing )

File: changelog_cheetah.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Device maintainer: Oscar Mariscal ()mariscal1019)
```
Comment (gwolf2u) [inline changelog_cheetah.txt:9]: fix additional open )

File: changelog_cheetah.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Highlights & Device Specific Changes:
+Build type: weekly
```
Comment (gwolf2u) [inline changelog_cheetah.txt:7]: Use capital W - "Weekly"

File: cheetah.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "Weekly",
+			"forum": "https://forum.xda-developers.com/t/rom-13-cheetah-crdroid-v9-3-07-04-2023.4574269/",
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://drive.google.com/drive/folders/1ZD5MFi0cd5xcjmx31hgIXYGc32KtS-ty?usp=share_link",
+			"paypal": "https://www.paypal.me/osmjr1019",
+			"telegram": "",
+			"dt": "https://github.com/osm1019/device_google_pantah",
```
Comment (gwolf2u) [inline cheetah.json:28]: use crDroid forks

File: cheetah.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "https://github.com/osm1019/device_google_gs201",
```
Comment (gwolf2u) [inline cheetah.json:29]: and here

File: cheetah.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+			"kernel": "https://github.com/osm1019/Kirisakura_Pantah"
```
Comment (gwolf2u) [inline cheetah.json:30]: and here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by osm1019** [comment]
ok will fix now

**Pull request closed without merging**